test/unit/models/users.test.js

```diff
@@ -425,6 +425,26 @@ describe("users", function () {
       const newUsername = await users.generateUniqueUsername("shubham", "sigdar");
       expect(newUsername).to.deep.equal("shubham-sigdar-2");
     });
+
+    it("should generate a unique username when no existing users are present", async function () {
+      const firstName = "john";
+      const lastName = "doe";
+
+      const whereStub = sinon.stub(userModel, "where").returnsThis();
+      const countStub = sinon.stub().returnsThis();
+      const getStub = sinon.stub().resolves({
+        data: () => ({ count: 0 }),
+      });
```

Comment:
Why are we mocking this?
We are using firestore emulator right

Reply:
fixed